templates/github/.github/workflows/publish.yml.j2

```diff
@@ -30,27 +30,21 @@ jobs:
     runs-on: "ubuntu-latest"
     needs:
       - "build"
-
-    env:
-      GITHUB_TOKEN: "{{ '${{ secrets.GITHUB_TOKEN }}' }}"
+    environment:
+      name: "pypi"
+      url: "https://pypi.org/p/{{ plugin_name | dash }}"
```

Comment:
This environment is the new GH thing for describing the "environment for a deployment" and the deployment in our case being the "upload to pypi".
You specify url here and not for the bindings. Is it needed or not?

Reply:
Sad, but I'll leave it to your discretion.